pkg/asset/installconfig/stock.go

```diff
@@ -51,12 +51,14 @@ func (s *StockImpl) EstablishStock(directory string, inputReader *bufio.Reader)
 	}
 	s.clusterID = &clusterID{}
 	s.emailAddress = &asset.UserProvided{
-		Prompt:      "Email Address:",
-		InputReader: inputReader,
+		Prompt:       "Email Address:",
+		DefaultValue: "a@b.c",
+		InputReader:  inputReader,
 	}
 	s.password = &asset.UserProvided{
-		Prompt:      "Password:",
-		InputReader: inputReader,
+		Prompt:       "Password:",
+		DefaultValue: "verysecure",
```

Comment:
I don't think we want a default password.  Users should have to supply something for this.

Reply:
+1